lib/ansible/module_utils/facts/hardware/linux.py

```diff
@@ -526,6 +526,25 @@ def get_mount_info(self, mount, device, uuids):
 
         return mount_size, uuid
 
+    @staticmethod
+    def _is_special_mount(device, fstype):
+        # The mounts not starting with slashes are zfs, network mounts and
+        # special pseudo file systems.
+        regular_filesystems = [
+            'ceph',
+            'cifs',
+            'fuse',
+            'glusterfs',
+            'gpfs',
+            'nfs',
+            'zfs'
+        ]
```

Comment:
In the conversation above, it was discussed to include network mounts and than wrap the processing code in timeout operator (granted, that's not in the PR yet).

Reply:
The problem is that timeouts are useless when a process is stuck in 'blocking' mode in the kernel, which is why we really want to avoid NFS in particular, but most 'network file systems' in general.

Reply:
In that case, I can just keep the original scope of the PR and enable ZFS (perhaps some of the non-network fuse mounts as well). #24644 could then probably be closed as `won't fix`.

Reply:
I think I'll need some help here. Can you give me an example of such option implementation, so I can do it the right way?